controller.js

```diff
@@ -1,12 +1,17 @@
-// const endpoint = require("./endpoints.json");
+const endpoints = require("./endpoints.json");
+const { fetchApiTopics } = require("./model");
 
-// exports.api = (req, res) => {
-//     res.status(200).send({ msg: "Working"})
-// }
+exports.api = (req, res) => {
+  res.status(200).send({ endpoints });
+};
 
-// exports.getApiTopics = (req, res) => {
-//     fetchApiTopics()
-//     .then((topics) => {
-//         res.status(200).send({ topics })
-//     })
-// }
\ No newline at end of file
+exports.getApiTopics = (req, res) => {
+    console.log("Getting topics")
+  fetchApiTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch((err) => {
```

Comment:
we don't just want to log the error - we should pass it to next so that the error handling middleware can handle the sending of the error response - use the NC notes on error handling to set up this error handling middleware when you're ready